README.md

```diff
@@ -161,6 +161,12 @@ We use this to redirect http to https in Rancher.
 
 We use this in CI to check security issues of Node.js dependencies.
 
+## kiwicom/oauth2-proxy
+
+- Base image: `buildpack-deps:jessie-curl`
+
+We use this as reverse proxy that provides authentication with Gitlab or other provider.
```

Comment:
Please mention that you got all this code from https://github.com/chauffer/dockerfiles/tree/master/oauth2-proxy

Reply:
Added, please review.